llm/deepseek-r1/deepseek-r1-vllm.yaml

```diff
@@ -0,0 +1,30 @@
+envs:
+  MODEL_NAME: deepseek-ai/DeepSeek-R1-Distill-Llama-8B
+  MAX_MODEL_LEN: 4096
+  HF_TOKEN: # TODO: Fill with your own huggingface token, or use --env to pass.
+
+resources:
+  accelerators: {L4:1, A10G:1, A10:1, A100:1, A100-80GB:1}
+  ports:
+    - 8000
+
+setup: |
+  conda activate vllm
+  if [ $? -ne 0 ]; then
+    conda create -n vllm python=3.10 -y
+    conda activate vllm
+  fi
+
+  pip install transformers
+  pip install vllm
```

Comment:
We should probably pin versions here since these examples tend to break otherwise.

Reply:
Also if it's not too hard - can we quickly compare the startup time with this `setup` vs directly using the vllm docker image without this `setup`?